client/webserver/site/src/html/forms.tmpl

```diff
@@ -145,6 +145,43 @@
     <span id="appPassSpan">[[[app_pw_reg]]]</span>
     [[[reg_confirm_submit]]]
   </div>
+  <div class="fs16 mt-4">
+    [[[provided_markets]]]
+    <table class="marketstable">
+      <thead>
+        <tr>
+          <th class="text-center">[[[asset_header]]]</th>
+          <th class="text-center">[[[fee_price_header]]]</th>
+          <th class="text-center">[[[fee_confs_header]]]</th>
+          <th class="text-center">[[[wallet_header]]]</th>
+        </tr>
+      </thead>
+      <tbody id="feeTableRows">
+        <tr id="feeRowTemplate">
+          <td>
+            <div class="d-flex justify-content-center align-items-center">
+              <img class="micro-icon mr-1" data-tmpl="asseticon">
+              <div data-tmpl="asset"></div>
+            </div>
+          </td>
+          <td class="text-center">
+            <div class="d-flex justify-content-center align-items-center">
+              <div data-tmpl="fee"></div>
+            </div>
+          </td>
+          <td class="text-center">
+            <div class="d-flex justify-content-center align-items-center">
+              <div data-tmpl="confs"></div>
+            </div>
+          </td>
+          <td class="text-center">
+            <button id="setupWallet" type="button" class="justify-content-center p-1 selected">[[[SetupWallet]]]</button>
```

Comment:
For discussion:

![image](https://user-images.githubusercontent.com/9373513/135311282-64049469-84ac-492e-af5a-e40f38d53922.png)

Ideally we would have the rows be shorter (less vertical space), and I think we could do that with buttons that just said "Choose" and "Setup", or perhaps just by making the buttons wider.  There is plenty of horizontal room because the values in the Fee and Confs columns don't take much space.